src/pages/challenges/index.tsx

```diff
@@ -54,7 +48,7 @@ const Challenges: FC = () => {
                 setFilteredChallenge(challengeMap);
```

Comment:
I think it would be if do this filtering in the `tranformResponse` of the RTK Query itself